extensions/ohif-i18n/src/locales/en/CineDialog.json

```diff
@@ -0,0 +1,8 @@
+{
+  "fps": "fps",
+  "Skip to first image": "Skip to first image",
+  "Previous image": "Previous, image",
+  "Play / Stop": "Play, / Stop",
+  "Next image": "Next, image",
+  "Skip to last image": "Skip, to last image"
```

Comment:
Remove the comma (same with the ones above)

Reply:
Fixed.